test/support/mocks.ex

```diff
@@ -13,3 +13,4 @@ Mox.defmock(CMS.Api.Mock, for: CMS.Api.Behaviour)
 Mox.defmock(MBTA.Api.Mock, for: MBTA.Api.Behaviour)
 Mox.defmock(RoutePatterns.Repo.Mock, for: RoutePatterns.Repo.Behaviour)
 Mox.defmock(OpenTripPlannerClient.Mock, for: OpenTripPlannerClient.Behaviour)
+Mox.defmock(Predictions.Repo.Mock, for: Predictions.Repo.Behaviour)
```

Comment:
Should probably separate out the `Repo.Mock`s and make sure they're alphabetized. I would label them too.